data_source_examples/slack_conversation_history/using_airtable.js/README.md

```diff
@@ -0,0 +1,66 @@
+# Bring Slack Conversation History into Airtable
+
+This code example can be used to import messages from a Slack conversation (public or private channel, DM, or multi-party DM) into an Airtable table. You can also schedule this script to run on a recurring to keep your table in Airtable up to date with the new messages from Slack.
+
+This code example is based on [the generic airtable.js upsert example](.../../../../../javascript/using_airtable.js/) and uses [`airtable.js`](https://github.com/airtable/airtable.js) to interact with the Airtable REST API and [`@slack/web-api`](https://slack.dev/node-slack-sdk/web-api) to interact with Slack's API.
+
+---
+
+The software made available from this repository is not supported by Formagrid Inc (Airtable) or part of the Airtable Service. It is made available on an "as is" basis and provided without express or implied warranties of any kind.
+
+---
+
+## Setup steps
+
+This section will walk you through setting up three components:
+
+- A. An Airtable base where this example script will store messages from the Slack conversation you specify
+- B. A custom Slack app and retrieving your Slack bot token, allowing you to retrieve messages from Slack
+- C. This script (which will pull messages from the Slack converstion, compare it to existing records in you base, and create-or-update records in your base so they mirror the latest messages in Slack)
+
+### A. Airtable Base Setup
+
+First, create a table in a base you have creator-level access to and create a table with the following fields: 'Channel ID + TS' (Single line text), 'Channel ID' (Single line text), 'TS' (Single line text), 'Last Edited TS' (Single line text), 'Type' (Single line text), and 'Subtype' (Single line text), 'Slack User ID' (Single line text), 'Message text' (Long text), 'Reply Count' (Number), 'Parent Message' (self-linking Linked Record), and 'Full Message Payload (JSON)' (Long text).
+
+- You may also want to add a formula field named 'TS (Human Readable)' with the formula `DATETIME_PARSE({TS}, 'X')` and another named 'Parent or Thread?' with the formula `IF({Parent Message},"Threaded reply","Parent message")`.
+- You can create a copy of a sample table with these fields [from here](https://airtable.com/shrB2653wGPc4KwoZ) by selecting "Use data" in the top right corner.
+- If you choose other field names, be sure to update the code in [index.js](./index.js) in the `convertSlackMessageToAirtableRecord` function.
```

Comment:
I'd recommend swapping the order:

First, create a table in a base you have creator-level access to. You can do so by creating a copy of this sample table (recommended). 

Alternatively you can create a table with the following fields: ....

As an aside: Perhaps this should be a markdown table? I must admit it took me longer than I'd like to admit to realize that TS === Timestamp, so short descriptions of what the fields do could be nice. You could also add Field Descriptions to the source table (and note in the Readme that those descriptions exist).

You could also link to the list of all field configurations in an "appendix" at the bottom of the Readme (you can link to other parts of the same readme), or a separate md file. Mostly - I like instructions to be as short as possible (with links to more details if needed) to avoid it being overwhelming, and my suggestion to use a markdown table might make it a lot more bulky.

Reply:
Great point!